cmd/operator/main.go

```diff
@@ -169,8 +173,11 @@ func parseFlags(fs *flag.FlagSet) {
 	fs.Var(&cfg.ThanosRulerSelector, "thanos-ruler-instance-selector", "Label selector to filter ThanosRuler Custom Resources to watch.")
 	fs.Var(&cfg.SecretListWatchSelector, "secret-field-selector", "Field selector to filter Secrets to watch")
 
+	// Auto GOMEMLIMIT Ratio
+	fs.Float64Var(&memlimitRatio, "auto-gomemlimit-ratio", 0.9, "The ratio of reserved GOMEMLIMIT memory to the detected maximum container or system memory. The value should be less than 0.0 setting to 0.0 and greater than 1.0 setting to 1.0. Default: 0.9")
```

Comment:
I'm having a really hard time understanding what is being said here 😬 

```
The value should be less than 0.0 setting to 0.0 and greater than 1.0 setting to 1.0
```

Could you clarify?

Reply:
If we're putting this behind a feature-gate, I think it makes sense to mention that here as well

Reply:
> I'm having a really hard time understanding what is being said here 😬
> 
> ```
> The value should be less than 0.0 setting to 0.0 and greater than 1.0 setting to 1.0
> ```
> 
> Could you clarify?

Change it to 
```
The value should be greater than 0.0 and less than 1.0. Default: 0.9
```